### PR TITLE
[plugin.video.hgtv] 3.0.3

### PR DESCRIPTION
**Pull request by learningit**

### CODE REVIEW

File: plugin.video.hgtv/resources/lib/scraper.py [hunk 1]
```diff
@@ -49,18 +49,23 @@ def getAddonMenu(self,url,ilist):
...
+       fanart = re.compile('property="og:image" content="(.+?)"',re.DOTALL).search(html)
+       if fanart != None:
+           fanart = fanart.group(1).replace('616.347.jpeg','1280.720.jpeg',1)
+       else:
+           fanart = thumb
        infoList['TVShowTitle'] = name
        infoList['Title']       = name
        infoList['Studio']      = 'HGTV'
        infoList['Plot'] = h.unescape(plot)
+       infoList['mediatype'] = 'tvshows'
```
Comment (phil65) [inline plugin.video.hgtv/resources/lib/scraper.py]: "tvshow", not "tvshows".

Comment (learningit) [inline plugin.video.hgtv/resources/lib/scraper.py]: Ok.Wasn't sure. I used this list for content types http://mirrors.kodi.tv/docs/python-docs/15.x-isengard/xbmcplugin.html#-setContent , where are the mediatypes listed?

Comment (phil65) [inline plugin.video.hgtv/resources/lib/scraper.py]: https://github.com/xbmc/xbmc/blob/master/xbmc/media/MediaType.h#L28-L39

Comment (phil65) [inline plugin.video.hgtv/resources/lib/scraper.py]: ...or the more obvious place (but incomplete): http://mirrors.kodi.tv/docs/python-docs/16.x-jarvis/xbmcgui.html#ListItem-setInfo ;)

**Comment by learningit** [comment]
Should be ok now. Thanks.

**Pull request closed without merging**